dedibox-console/account/how-to/terminate-dedibox-server.md

```diff
@@ -0,0 +1,31 @@
+---
+meta:
+  title: How to terminate a Dedibox server under Online console
+  description: Learn how to terminate a Dedibox server under online console.
+content:
+  h1: How to terminate a Dedibox server
+  paragraph: This guide explains how to terminate a Online Dedibox server.
+tags: dedibox terminate server online console
+dates:
+  validation: 2024-12-12
+categories:
+  - dedibox-console
+---
+
+<Macro id="requirements" />
+- A Dedibox account logged into the [console](https://console.online.net)
+- An active Dedibox server
+
+### How to terminate your Web Hosting plan
```

Comment:
This should be an H2 (##)

Also, should this talk about web hosting plan, or should it be "How to terminate a Dedibox server" ?

Reply:
Fixed, was a misreading